resetpass/index.html

```diff
@@ -20,26 +20,85 @@
       <script src="../../js/html5shiv.min.js"></script>
       <script src="../../js/respond.min.js"></script>
       <![endif]-->
-      
+
       <script src="../../js/angular.min.js"></script>
       <script src="../../js/angular-route.min.js"></script>
       <script src="../../js/loklak.js"></script>
       <script src="../../js/jquery.min.js"></script>
       <script src="js/resetpass.js"></script>
+=======
+  <meta name="Content-Language" content="en, english">
+  <meta name="author" content="@shivenmian">
+  <link rel="icon" type="image/png" href="../../artwork/favicon.png">
+  <title>Reset Loklak Password</title>
+  <link href="../../css/bootstrap.min.css" rel="stylesheet">
+  <link href="../../css/dashboard.css" rel="stylesheet">
+  <link href="../../css/loklak.css" rel="stylesheet">
+  <link href="css/style.css" rel="stylesheet">
+
+  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+<script src="../../js/html5shiv.min.js"></script>
+<script src="../../js/respond.min.js"></script>
+<![endif]-->
+
+<script src="../../js/angular.min.js"></script>
+<script src="../../js/angular-route.min.js"></script>
+<script src="../../js/loklak.js"></script>
+<script src="../../js/jquery.min.js"></script>
+<script src="js/resetpass.js"></script>
+>>>>>>> 0f6699c... Fixed html lint errors
     </head>
 
     <body>
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```

Comment:
Line is longer than allowed. (144 > 129)

*LineLengthBear, severity NORMAL, section `default`.*